addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,311 @@
+account      / account.chart.template   / company_id (many2one)         : DEL relation: res.company
+# DONE: post-migration: tried to fill all of them them using the company
+
+account      / account.fiscal.year      / company_id (many2one)         : NEW relation: res.company, required: required, req_default: function
+account      / account.fiscal.year      / date_from (date)              : NEW required: required
+account      / account.fiscal.year      / date_to (date)                : NEW required: required
+account      / account.fiscal.year      / name (char)                   : NEW required: required
+# NOTHING TO DO (new model)
+
+account      / account.group            / parent_left (integer)         : DEL
+account      / account.group            / parent_path (char)            : NEW 
+account      / account.group            / parent_right (integer)        : DEL 
+# DONE: post-migration: call _parent_store_compute() method
+
+account      / account.incoterms        / active (boolean)              : NEW
+account      / account.incoterms        / code (char)                   : NEW required: required
+account      / account.incoterms        / name (char)                   : NEW required: required
+# NOTHING TO DO (rename done in apriori)
+
+account      / account.invoice          / activity_date_deadline (date) : now a function
+account      / account.invoice          / incoterm_id (many2one)        : previously in module l10n_be_intrastat
+account      / account.invoice          / message_last_post (datetime)  : DEL 
+account      / account.invoice          / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+account      / account.invoice          / reference_type (selection)    : DEL required: required, selection_keys: function, req_default: function
+account      / account.invoice          / source_email (char)           : NEW
+account      / account.invoice          / state (selection)             : selection_keys is now '['cancel', 'draft', 'in_payment', 'open', 'paid']' ('['cancel', 'draft', 'open', 'paid']')
+account      / account.invoice          / vendor_bill_id (many2one)     : NEW relation: account.invoice
+# NOTHING TO DO
+
+account      / account.invoice.line     / display_type (selection)      : NEW selection_keys: ['line_note', 'line_section']
+# DONE: post-migration: filled as 'line_section' from layout_category_id
+
+account      / account.invoice.line     / uom_id (many2one)             : relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+account      / account.invoice.tax      / analytic_tag_ids (many2many)  : NEW relation: account.analytic.tag
+# NOTHING TO DO
+
+account      / account.journal          / alias_id (many2one)           : NEW relation: mail.alias
+# DONE: post-migration: filled for all journals fo type 'purchase'
+
+account      / account.journal          / bank_statements_source (selection): selection_keys is now 'function' ('['manual', 'undefined']')
```

Comment:
`manual` doesn't exist anymore, so you have to update the journals with this value to `undefined`.